share/completions/rsync.fish

```diff
@@ -16,30 +32,41 @@ complete -c rsync -l suffix -d "Backup suffix (default ~ w/o --backup-dir)"
 complete -c rsync -s u -l update -d "Skip files that are newer on the receiver"
 complete -c rsync -l inplace -d "Update destination files in-place"
 complete -c rsync -l append -d "Append data onto shorter files"
+complete -c rsync -l append-verify -d "--append w/old data in file checksum"
 complete -c rsync -s d -l dirs -d "Transfer directories without recursing"
+complete -c rsync -l mkpath -d "Create the destination's path component"
 complete -c rsync -s l -l links -d "Copy symlinks as symlinks"
 complete -c rsync -s L -l copy-links -d "Transform symlink into referent file/dir"
 complete -c rsync -l copy-unsafe-links -d "Only \"unsafe\" symlinks are transformed"
 complete -c rsync -l safe-links -d "Ignore symlinks that point outside the tree"
+complete -c rsync -l munge-links -d "Munge symlinks to make them safe & unusable"
 complete -c rsync -s k -l copy-dirlinks -d "Transform symlink to dir into referent dir"
 complete -c rsync -s K -l keep-dirlinks -d "Treat symlinked dir on receiver as dir"
 complete -c rsync -s H -l hard-links -d "Preserve hard links"
 complete -c rsync -s p -l perms -d "Preserve permissions"
 complete -c rsync -s E -l executability -d "Preserve executability"
+complete -c rsync -l chmod -d "Change destination permissions"
 complete -c rsync -s A -l acls -d "Preserve ACLs (implies -p) [non-standard]"
 complete -c rsync -s X -l xattrs -d "Preserve extended attrs (implies -p) [n.s.]"
-complete -c rsync -l chmod -d "Change destination permissions"
 complete -c rsync -s o -l owner -d "Preserve owner (super-user only)"
 complete -c rsync -s g -l group -d "Preserve group"
 complete -c rsync -l devices -d "Preserve device files (super-user only)"
 complete -c rsync -l specials -d "Preserve special files"
 complete -c rsync -s D -d "Same as --devices --specials"
-complete -c rsync -s t -l times -d "Preserve times"
+complete -c rsync -s t -l times -d "Preserve modification times"
+complete -c rsync -s U -l atimes -d "Preserve access (use) times"
+complete -c rsync -l open-noatime -d "Avoid changing the atime on opened files"
+complete -c rsync -l crtimes -d "Preserve create times (newness)"
```

Comment:
This is the first time I'm reading the term "newness" for creation time, and I don't think it helps to clarify anything. "create time" or "creation time" is already perfectly clear.

Reply:
I agree, did a quick web search now and could not find other mentions... seems to be only in this man page. 
In `stat` it is called birth or `btime` in the struct, so I have changed it.